app/helpers/alchemy/admin/elements_helper.rb

```diff
@@ -105,8 +105,8 @@ def element_array_for_options(e, object_method, cell = nil)
       #
       def update_essence_select_elements(page, element)
         elements = page.elements.not_trashed.joins(:contents)
-          .where("alchemy_contents.element_id != #{element.id}")
-          .where("alchemy_contents.essence_type" => "Alchemy::EssenceSelect")
+          .where("#{Content.table_name}.element_id != #{element.id}")
+          .where("#{Content.table_name}.essence_type" => "Alchemy::EssenceSelect")
```

Comment:
Line is too long. [82/80]<br>Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.